### PR TITLE
Updates 1

### PR DESCRIPTION
**Pull request by martacki**
- adding a select box for countries on page 3

### CODE REVIEW

File: app/pages/1_Scenario_comparison.py [hunk 1]
```diff
@@ -37,7 +37,8 @@ def main():
     tech_map = dict(map(reversed, carriers_map.items()))
 
     ###### first dropdown plotting n.statistics #####
-    params = list(network_map.values())[0].statistics().columns
+    remove_params = set(["Capacity Factor", "Supply", "Withdrawal"])
+    params = list(set(list(network_map.values())[0].statistics().columns)-remove_params)
```
Comment (ekatef) [inline app/pages/1_Scenario_comparison.py:41]: Agree with `Supply` and `Withdrawal`, but would rather sacrifice with something like `Revenue` than with `Capacity Factor`. I feel the latter one is quite relevant for a high-level overview.

File: app/pages/3_Country_operation.py [hunk 2]
```diff
@@ -105,7 +105,7 @@ def get_colors_map():
 with suppl_col:
     choices = upd_dict
     res = st.selectbox(
-        "Resolution",
+        "Country",
```
Comment (ekatef) [inline app/pages/3_Country_operation.py:108]: As we discussed, that may be important to have functionality to use time-aggregation as a kind of smoothing filter. So, I think `Resolution` box should be back.

**Comment by ekatef** [comment]
Thanks for the contribution @martacki and sorry for miscommunication. Have added a couple of comments to the code, mainly just writing down what we have discussed.

I'll do my best to accommodate the PR, but git shows some conflicts, unfortunately. So, it may get tricky.

An additional comment: as you could see, currently the helper files have a lot of code duplication and definitely would benefit from refactoring. That is not streamlit feature, just a need to make some code polishing. I'd try to quickly fix basic functionality, and after that make `system_operation_prerun.py` look nicer. Shall try to keep you informed, if I'll plan to go for refactoring.

**Comment by martacki** [comment]
> Thanks for the contribution @martacki and sorry for miscommunication. Have added a couple of comments to the code, mainly just writing down what we have discussed.
> 
> I'll do my best to accommodate the PR, but git shows some conflicts, unfortunately. So, it may get tricky.
> 
> An additional comment: as you could see, currently the helper files have a lot of code duplication and definitely would benefit from refactoring. That is not streamlit feature, just a need to make some code polishing. I'd try to quickly fix basic functionality, and after that make `system_operation_prerun.py` look nicer. Shall try to keep you informed, if I'll plan to go for refactoring.

Let's close and focus on the other PRs instead, to save time

**Comment by martacki** [comment]
> Thanks for the contribution @martacki and sorry for miscommunication. Have added a couple of comments to the code, mainly just writing down what we have discussed.
> 
> I'll do my best to accommodate the PR, but git shows some conflicts, unfortunately. So, it may get tricky.
> 
> An additional comment: as you could see, currently the helper files have a lot of code duplication and definitely would benefit from refactoring. That is not streamlit feature, just a need to make some code polishing. I'd try to quickly fix basic functionality, and after that make `system_operation_prerun.py` look nicer. Shall try to keep you informed, if I'll plan to go for refactoring.

Let's close and focus on the other PRs instead, to save time

**Pull request closed without merging**